lib/node_modules/@stdlib/stats/base/dists/studentized-range/cdf/benchmark/benchmark.js

```diff
@@ -21,27 +21,35 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pkg = require( './../package.json' ).name;
 var cdf = require( './../lib' );
+var uniform = require('@stdlib/random/base/uniform')
 
```

Comment:
```suggestion
var Float64Array = require( '@stdlib/array/float64' );
```

This applies throughout the PR